src/3d/qgsmeshlayer3drenderer.h

```diff
@@ -0,0 +1,89 @@
+/***************************************************************************
+  qgsmeshlayer3drenderer.h
+  ------------------------
+  Date                 : January 2019
+  Copyright            : (C) 2019 by Peter Petrik
+  Email                : zilolv at gmail dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSMESHLAYER3DRENDERER_H
+#define QGSMESHLAYER3DRENDERER_H
+
+#define SIP_NO_FILE
```

Comment:
Vector layer 3D renderer now has (experimental) Python support, so best to enable Python API for mesh 3d renderer too